docs/configuration.asciidoc

```diff
@@ -426,7 +426,9 @@ If you want to dynamically change the status of the agent, use <<config-recordin
 [[config-instrument]]
 ==== `instrument` (added[1.0.0,Changing this value at runtime is possible since version 1.15.0])
 
-A boolean specifying if the agent should instrument the application to collect performance metrics for the app. When set to false, Elastic APM will not affect your application at all.
+A boolean specifying if the agent should instrument the application to collect traces for the app.
+ When set to `false`, Elastic APM will not apply bytecode instrumentation, thus it would have no effect on 
```

Comment:
Technically, the agent still does some bytecode instrumentation but only for things related to manual APIs, such as the public API, the OT bridge, and Micrometer.